src/main/java/backend/resource/TurboIssue.java

```diff
@@ -390,6 +392,23 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return getMatchedIssues(issues, query).stream().findFirst();
     }
 
+    /**
+     * @param queries
+     * @return number of matches to a list of query
+     */
+    private int getNumberOfMatchingQuery(List<String> queries) {
+        return (int) queries.stream()
```

Comment:
We can use [toIntExact](http://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#toIntExact-long-) here.

Just realised we have `Utility. safeLongToInt` also. That can probably be removed now if it does the same thing.